src/pudl/transform/epacems.py

```diff
@@ -59,22 +126,28 @@ def fix_up_dates(df: pd.DataFrame, plant_utc_offset: pd.DataFrame) -> pd.DataFra
     # deprecated, but the PyArrow schema stores this data as UTC. See:
     # https://numpy.org/devdocs/reference/arrays.datetime.html#basic-datetimes
     df["operating_datetime_utc"] = df["op_datetime_naive"] - df["utc_offset"]
-    del df["op_date"], df["op_hour"], df["op_datetime_naive"], df["utc_offset"]
+    del (
+        df["op_date"],
+        df["op_hour"],
+        df["op_datetime_naive"],
+        df["utc_offset"],
+        df["plant_id_combined"],
```

Comment:
if you previously renamed `plant_id_eia` to `plant_id_combined` doesn't this drop also drop the original `plant_id_eia`?

Reply:
I thiiink this is based on the assumption that `plant_id_eia` previously existed in the CEMS data which is doesn't (as explained above). If not, lmk!

Reply:
Also, there is no more `plant_id_combined` anymore!